com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl

```diff
@@ -322,7 +322,7 @@ half AdditionalLightRealtimeShadow(int lightIndex, float3 positionWS, half3 ligh
 
     int shadowSliceIndex = shadowParams.w;
 
-    UNITY_BRANCH
```

Comment:
The `UNITY_BRANCH` was causing a 
`Error: use of potentially uninitialized variable (AdditionalLightRealtimeShadow)`
error to appear.